docs/index.rst

```diff
@@ -94,12 +94,20 @@ Start Example
       cd cp-demo
       git checkout |release_post_branch|
 
-#. From the ``cp-demo`` directory, start the entire example by running a single command that generates the keys and certificates, brings up the Docker containers, and configures and validates the environment. This takes approximately 10 minutes to complete.
+#. From the ``cp-demo`` directory, start the entire example by running a single command that generates the keys and certificates, brings up the Docker containers, and configures and validates the environment.
 
-   .. sourcecode:: bash
+   - The first time you run ``cp-demo``, the startup takes approximately 7 minutes to complete.
```

Comment:
Re text that describes time-expectations ... since these will vary constantly over time and be subject to optimisations, and may be hardware-dependant ... this change is fine for now, but if we are back here again, maybe we can come up with something a little vaguer ...

> the startup time will depend on your hardware and whether Docker images are already cached on your machine, but expect startup to take more than 5 minutes.

Reply:
@javabrett Text is now vaguer